javascript/integration-tests/rpc/creditcoin.spec.ts

```diff
@@ -2,210 +2,169 @@
-        // or references other unknown data types
-        friendly__Event: {
-          _enum: {
-            ctcTransfer: {
-              from: 'AccountId',
-              to: 'AccountId',
-              amount: String,
-            },
-            ctcDeposit: {
-              into: 'AccountId',
-              amount: String,
-            },
-            ctcWithdraw: {
-              from: 'AccountId',
-              amount: String,
-            },
-            rewardIssued: {
-              to: 'AccountId',
-              amount: String,
-            },
-            addressRegistered: {
-              address_id: 'AddressId<Hash>',
-              address: 'Address<AccountId>',
-            },
-            transferRegistered: {
-              transfer_id: 'TransferId<Hash>',
-              transfer: 'Transfer<AccountId, BlockNumber, Hash>',
-            },
-            transferVerified: {
-              transfer_id: 'TransferId<Hash>',
-              transfer: 'Transfer<AccountId, BlockNumber, Hash>',
-            },
-            transferProcessed: {
-              transfer_id: 'TransferId<Hash>',
-              transfer: 'Transfer<AccountId, BlockNumber, Hash>',
-            },
-            askOrderAdded: {
-              ask_id: 'AskOrderId<BlockNumber, Hash>',
-              ask: 'AskOrder<AccountId, BlockNumber, Hash, Moment>',
-            },
-            bidOrderAdded: {
-              bid_id: 'BidOrderId<BlockNumber, Hash>',
-              bid: 'BidOrder<AccountId, BlockNumber, Hash, Moment>',
-            },
-            offerAdded: {
-              offer_id: 'OfferId<BlockNumber, Hash>',
-              offer: 'Offer<AccountId, BlockNumber, Hash>',
-            },
-            dealOrderAdded: {
-              deal_id: 'DealOrderId<BlockNumber, Hash>',
-              deal: 'DealOrder<AccountId, BlockNumber, Hash, Moment>',
-            },
-            dealOrderFunded: {
-              deal_id: 'DealOrderId<BlockNumber, Hash>',
-              deal: 'DealOrder<AccountId, BlockNumber, Hash, Moment>',
-            },
-            dealOrderClosed: {
-              deal_id: 'DealOrderId<BlockNumber, Hash>',
-              deal: 'DealOrder<AccountId, BlockNumber, Hash, Moment>',
-            },
-            loanExempted: {
-                deal_id: 'DealOrderId<BlockNumber, Hash>',
-                exempt_transfer_id: 'TransferId<Hash>',
-            },
-            legacyWalletClaimed: {
-              new_account_id: 'AccountId',
-              legacy_sighash: 'LegacySighash',
-              claimed_balance: String,
-            },
-          }
-        },
-      },
-      rpc: {
-        creditcoin: {
-          getEvents: {
-            params: [
-              {
-                name: 'at',
-                type: 'Hash',
-                isOptional: true
-              }
-            ],
-            type: 'Vec<friendly__Event>'
-          }
-        }
-      }
+    let api: ApiPromise;
```

Comment:
This looks like formatting changes only. Can you revert them b/c they aren't related to the current PR and will conflict with a lot more changes from #125.